### PR TITLE
ESP32: Fix lock-app protos

### PR DESCRIPTION
**Pull request by sweetymhaiske**
#### Problem
Compilation of lock-app fails when Pigweed is enabled.

#### Change overview
Made the necessary changes in the lighting service proto.

#### Testing
Compiled the app successfully and tested using pigweed HDLC console.
However, for a successful compilation, I have added the necessary changes of this PR #8129.

### CODE REVIEW

**Review by msandstedt — APPROVED** [review]

**Review by saurabhst — APPROVED** [review]

**Review by rgoliver — COMMENTED** [review]
TLDR: Switi and I chatted, I don't think this CL is needed.

I don't believe this is a problem compiling but using the hdlc-console included with Pigweed, the hdlc console doesn't support providing dependent protos on the command line. For the lighting app I added a custom console which bundles all the required protos together which both fixes this and makes it easier to use/deploy.

Rather then creating a new console for every app, I am going to create a single chip-rpc-console which bundles all the possible chip protos into one tool (including this locking one). I have this new console working now, and just working through cleanup and testing across the different platforms/apps, it should be up today or early next week.

**Comment by dhrishi** [comment]
@sweetymhaiske Please close this if it is not applicable now (as per the comments from @rgoliver )

**Comment by sweetymhaiske** [comment]
RPC with lock-app fails now, waiting for the custom console.
